pennylane/ops/op_math/__init__.py

```diff
@@ -43,6 +43,7 @@
     ~CompositeOp
     ~Controlled
     ~ControlledOp
+    ~ControlledQubitUnitary
```

Comment:
I don't necessarily think we need to include it here, but maybe that's my preconceived notions of ControlledQubitUnitary as a "normal" operation. My intuition says to remove it and keep it in the same category as "normal" ops, but I'm fine either way.

Reply:
So would you be inclined to just not list it here, even though it lives in op_math?

Reply:
Maybe we can add a new section for operations like `ControlledQubitUnitary`? Especially since we can repeat this restructuring for more "basic" operations.